bindings/wasm/src/xlsx.rs

```diff
@@ -0,0 +1,39 @@
+use ironcalc::{
+    base::Model as BaseWorkbookModel, export::save_xlsx_to_writer, import::load_from_xlsx_bytes,
+};
+use std::io::{BufWriter, Cursor, Write};
+use wasm_bindgen::prelude::{wasm_bindgen, JsError};
+
+fn to_js_error(error: String) -> JsError {
+    JsError::new(&error.to_string())
+}
+
+#[wasm_bindgen(js_name = fromXLSXBytes)]
+pub fn from_xlsx_bytes(
+    bytes: &[u8],
+    name: &str,
+    locale: &str,
+    timezone: &str,
+) -> Result<Vec<u8>, JsError> {
+    let workbook = load_from_xlsx_bytes(bytes, name, locale, timezone)
+        .map_err(|e| to_js_error(e.to_string()))?;
+    let base_model =
+        BaseWorkbookModel::from_workbook(workbook).map_err(|e| to_js_error(e.to_string()))?;
+    Ok(base_model.to_bytes())
+}
+
+#[wasm_bindgen(js_name = toXLSXBytes)]
+pub fn to_xlsx_bytes(bytes: &[u8]) -> Result<Vec<u8>, JsError> {
+    let workbook = BaseWorkbookModel::from_bytes(bytes).map_err(to_js_error)?;
+    let mut writer = BufWriter::new(Cursor::new(Vec::new()));
+    save_xlsx_to_writer(&workbook, &mut writer).map_err(|e| to_js_error(e.to_string()))?;
+    writer.flush().map_err(|e| to_js_error(e.to_string()))?;
+    Ok(writer
+        .into_inner()
+        .map_err(|e| to_js_error(e.to_string()))?
+        .into_inner())
+}
+
+fn main() {
+    // This is required for cargo to compile this as a binary
+}
```

Comment:
I'm a bit curious about this, why didn't you go the same way as the other build? Note in Cargo you can have different build targets. See for instance in IronCalc:

https://github.com/ironcalc/IronCalc/blob/main/base/Cargo.toml#L24-L27

https://github.com/ironcalc/IronCalc/blob/main/base/src/model.rs#L38-L64

Maybe that would make things easier? We just have two builds with different features?

Reply:
This is so we can have one crate, but two wasm files outputted.

If there isn't a main declaration, cargo would just treat it as a library and won't turn it into an artifact.

I'm pretty new to rust, so if you have a preferred method, I would defer to you here!